en/ability.json

```diff
@@ -193,83 +193,83 @@
-    "description": "The Pokémon may pick up the item an opposing Pokémon held during a battle."
+    "description": "The Pokémon will attempt to steal an item that an opposing Pokémon held after a battle is won, even if it did not participate."
   },
   "truant": {
     "name": "Truant",
-    "description": "The Pokémon can’t use a move if it had used a move on the previous turn."
+    "description": "Each time the Pokémon uses a move, it spends the next turn loafing around."
   },
   "hustle": {
     "name": "Hustle",
-    "description": "Boosts the Attack stat, but lowers accuracy."
+    "description": "Boosts the Pokémon’s Attack stat by 50%, but lowers the accuracy of physical moves by 20%."
   },
   "cuteCharm": {
     "name": "Cute Charm",
-    "description": "Contact with the Pokémon may cause infatuation."
+    "description": "Contact with the Pokémon has a 30% chance to infatuate the attacker."
   },
   "plus": {
     "name": "Plus",
-    "description": "Boosts the Sp. Atk stat of the Pokémon if an ally with the Plus or Minus Ability is also in battle."
+    "description": "Boosts the Sp. Atk stat of the Pokémon by 50% if an ally with the Plus or Minus Ability is also in battle."
   },
   "minus": {
     "name": "Minus",
-    "description": "Boosts the Sp. Atk stat of the Pokémon if an ally with the Plus or Minus Ability is also in battle."
+    "description": "Boosts the Sp. Atk stat of the Pokémon by 50% if an ally with the Plus or Minus Ability is also in battle."
   },
   "forecast": {
     "name": "Forecast",
     "description": "The Pokémon transforms with the weather to change its type to Water, Fire, or Ice."
   },
   "stickyHold": {
     "name": "Sticky Hold",
-    "description": "Items held by the Pokémon are stuck fast and cannot be removed by other Pokémon."
+    "description": "The Pokémon’s held items cling to its sticky body and cannot be removed by other Pokémon."
   },
   "shedSkin": {
     "name": "Shed Skin",
-    "description": "The Pokémon may heal its own status conditions by shedding its skin."
+    "description": "The Pokémon has a 1/3 chance to cure its own status conditions by shedding its skin."
   },
   "guts": {
     "name": "Guts",
-    "description": "It’s so gutsy that having a status condition boosts the Pokémon’s Attack stat."
+    "description": "It’s so gutsy that having a status condition boosts the Pokémon’s Attack stat by 50%."
   },
   "marvelScale": {
     "name": "Marvel Scale",
-    "description": "The Pokémon’s marvelous scales boost the Defense stat if it has a status condition."
+    "description": "The Pokémon’s marvelous scales boost its Defense stat by 50% if it has a status condition."
   },
   "liquidOoze": {
     "name": "Liquid Ooze",
-    "description": "The oozed liquid has a strong stench, which damages attackers using any draining move."
+    "description": "The strong stench of the Pokémon’s oozed liquid damages attackers that use HP-draining moves instead of healing them."
```

Comment:
```suggestion
    "description": "The strong stench of the Pokémon’s oozed liquid damages attackers that use HP-draining moves instead of the attacker being healed."
```